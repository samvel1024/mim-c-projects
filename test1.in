c4
d3
f2
e2
d2
f4
