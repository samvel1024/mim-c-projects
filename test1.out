--------1
--------2
--------3
---BC---4
---CB---5
--------6
--------7
--------8
abcdefgh
c4 c3 0
--------1
--------2
--B-----3
--CBC---4
---CB---5
--------6
--------7
--------8
abcdefgh
d3 e3 -2
--------1
--------2
--BBB---3
--CCB---4
---CB---5
--------6
--------7
--------8
abcdefgh
f2 b4 -4
--------1
-----C--2
--BBC---3
-BBBB---4
---CB---5
--------6
--------7
--------8
abcdefgh
? -4
--------1
-----C--2
--BBC---3
-BBBB---4
---CB---5
--------6
--------7
--------8
abcdefgh
d2 e2 -4
--------1
---CBC--2
--BBB---3
-BBCB---4
---CB---5
--------6
--------7
--------8
abcdefgh
f4 g4 -6
--------1
---CBC--2
--BBC---3
-BBBBBB-4
---CB---5
--------6
--------7
--------8
abcdefgh
