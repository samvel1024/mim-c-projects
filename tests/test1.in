8
1 3 2 1 2 5 5 3