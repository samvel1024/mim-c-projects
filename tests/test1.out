10111
a__b_
_c___
d_$__
____f
__g__
_h_7_
j___k
